python/tests/unit/connectors/test_function_choice_behavior.py

```diff
@@ -13,7 +13,9 @@
     DEFAULT_MAX_AUTO_INVOKE_ATTEMPTS,
     FunctionChoiceBehavior,
     FunctionChoiceType,
+    _combine_filter_dicts,
```

Comment:
Can we remove the prefix `_` from this function as is no longer a private function?

Reply:
This is still a private function which is used as part of the `FunctionChoiceBehavior` class. It's only being imported here for explicit testing coverage.